src/data/guilty_gear_xrd_rev_2/johnny.yml

```diff
@@ -0,0 +1,76 @@
+---
+name: Johnny
+moves:
+  normals:
+    - 5P
+    - 5K
+    - c.S
+    - f.S
+    - 5H
+    - 5D
+    - 6P
+    - 6K
+    - 6H
+    - 2P
+    - 2K
+    - 2S
+    - 2H
+    - 2D
+    - j.P
+    - j.K
+    - j.S
+    - j.H
+    - j.D
+  specials:
+    - name: Glitter is Gold
+      inputs:
+         - 236H
+    - name: Baccus Sigh
+      inputs:
+        - 41236P
+    - name: Mist Stance
+      inputs:
+        - '214[P]'
+        - '214[K]'
+        - '214[S]'
+        - 'j.214[P]'
+        - 'j.214[K]'
+        - 'j.214[S]'
+    - name: Cancel
+      inputs: H
+      conditions: Mist Stance
+    - name: Mist Step
+    - inputs:
+      - 66
+      - 44
+      conditions: Ground Mist Stance
+    - name: High Mist Finer
```

Comment:
The high/mid/low Mist Finer attacks are technically on the release of the button held to enter mist stance.

Let's change the inputs for them to be `]P[`, `]K[` and `]S[`. And make them `conditions: Mist Stance`. Same for the air versions, with `conditions: Air Mist Finer stance`.

Reply:
So the entire input of High Mist Finer (`214P`) is executed immediately upon release of whatever attack button is being held to maintain Mist Stance?

Reply:
```
08:16 <@achvatal> so for mist finer
08:16 <@achvatal> 214keydown puts him into mist stance
08:17 <@achvatal> while holding the button, he can shimmy around and whatnot
08:17 <@achvatal> then when he lets go of whatever button he pressed to enter the stance, he does mist finer attack
08:17 <adamprice> ohhhh, and then 'key-up' -> attack triggers
08:17 <adamprice> got it
08:18 <adamprice> okay that's weird
08:18 <@achvatal> but yeah, if you just do regular 214button, it's the ~same thing
08:18 <adamprice> :|
```

Reply:
updated